bitnami/thanos/values.yaml

```diff
@@ -408,7 +408,7 @@ query:
       ## @param query.grpc.server.tls.cert TLS Certificate for GRPC server - ignored if existingSecret is provided
       ## @param query.grpc.server.tls.key TLS Key for GRPC server - ignored if existingSecret is provided
       ## @param query.grpc.server.tls.ca TLS CA to verify clients against - ignored if existingSecret is provided
-      ## @param query.grpc.server.tls.existingSecret Existing secret containing your own TLS certificates
+      ## @param query.grpc.server.tls.existingSecret Existing secret containing your own TLS certificates. In the case that they are not named ca-cert, tls-cert and tls-key you will need to use the keyMapping accordingly.
```

Comment:
Maybe also include a note to tell the user to check for an example in the `values.yaml` file?

For instance, if the user is reading the chart documentation in ArtifactHub, these example comments are not exposed via the main documentation, so being explicit there's an example available in `values.yaml` might help.